code/__HELPERS/_errors.dm

```diff
@@ -0,0 +1,20 @@
+///Tosses an error into the exception list with a manual error name. Use ONLY when a stack_trace call is pointless.
```

Comment:
Give an example of when a stack_trace call is pointless, I have no idea what this means.

Reply:
In this PR its used in `icon_exists()`. The stack trace of the proc is irrelevant, since all the proc does is check a file for an icon_state. It's a nice way to throw a noticable error that also doesn't have a metric assload of useless information/waste a `CRASH()`

Reply:
My point is the PR body is not going to merged into your PR, and your comment right now is useless.

Include a more detailed comment in the code itself.

Reply:
Ah okay.